sdk/typescript/test/e2e/dwallet-mpc.test.ts

```diff
@@ -6,31 +6,44 @@ import { getFaucetHost, requestSuiFromFaucetV1 } from '@mysten/sui/faucet';
 import { Ed25519Keypair } from '@mysten/sui/keypairs/ed25519';
 import { beforeEach, describe, it } from 'vitest';
 
-import { createDWallet } from '../../src/dwallet-mpc/dkg';
-import { getOrCreateClassGroupsKeyPair } from '../../src/dwallet-mpc/encrypt-user-share';
-import { Config, delay, mockedProtocolPublicParameters } from '../../src/dwallet-mpc/globals';
+import {
+	createDKGFirstRoundOutputMock,
+	createDWallet,
+	dkgSecondRoundMoveCall,
+} from '../../src/dwallet-mpc/dkg';
+import {
+	checkpointCreationTime,
+	Config,
+	delay,
+	getDWalletSecpState,
+	mockedProtocolPublicParameters,
+} from '../../src/dwallet-mpc/globals';
+import { dkgFirstRoundMock } from '../../src/dwallet-mpc/mocks';
 
 const fiveMinutes = 5 * 60 * 1000;
 describe('Test dWallet MPC', () => {
 	let conf: Config;
 
 	beforeEach(async () => {
-		const keypair = Ed25519Keypair.generate();
+		const keypair = Ed25519Keypair.deriveKeypairFromSeed('0x1');
+		const dWalletSeed = new Uint8Array(32).fill(8);
+		const encryptedSecretShareSigningKeypair = Ed25519Keypair.deriveKeypairFromSeed(
+			Buffer.from(dWalletSeed).toString('hex'),
+		);
 		const address = keypair.getPublicKey().toSuiAddress();
 		const suiClient = new SuiClient({ url: getFullnodeUrl('localnet') });
 		await requestSuiFromFaucetV1({
 			host: getFaucetHost('localnet'),
 			recipient: address,
 		});
-		const dWalletSeed = new Uint8Array(32);
-		crypto.getRandomValues(dWalletSeed);
 
 		conf = {
-			keypair,
+			suiClientKeypair: keypair,
 			client: suiClient,
 			timeout: fiveMinutes,
 			ikaConfig: require('../../../../ika_config.json'),
```

Comment:
not part of this PR, but I didn't notice, move this require to the top, and see if you can use some anchor to root dir in order to derive the path from it.